docs/REST API/rest-api.md

```diff
@@ -7123,12 +7136,125 @@ Deletes the session specified by ID. When the user associated with the session n
 
 ## Software
 
-- [List all software](#list-all-software)
-- [Count software](#count-software)
+- [List software titles](#list-software-titles)
```

Comment:
@marko-lisica maybe we trim this to "List software" this matches what we're doing in the UI? (no mention of "titles")

If we go with this change, reminder to update the section title as well: https://github.com/fleetdm/fleet/pull/14831/files#diff-7246bc304b15c8865ed8eaa205e9c244d0a0314e4bae60cf553dc06147c38b64R7144